tests/profile/ProfileUiTest.php

```diff
@@ -1,13 +1,12 @@
 <?php
 
-class ProfileUiTest extends PHPUnit_Framework_TestCase
+require_once('AbstractProfileTest.php');
```

Comment:
I started with trying to use namespaces, but I've never used them before and after 30 minutes of wondering the `AbstratProfileTest` class couldn't be found I gave up and resorted to using what I know. I tried adding vendor/autoload.php to the bootstrap for phpunit and that didn't work.